docs/applications/development/wildfly.md

```diff
@@ -0,0 +1,483 @@
+        cp $WILDFLY_DIR/bin/standalone.conf $WILDFLY_DIR/bin/standalone-org.conf
+
+        echo "Configuring application server..."
+        sed -i -e 's,<deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="5000"/>,<deployment-scanner path="deployments" relative-to="jboss.server.base.dir" scan-interval="5000" deployment-timeout="'$WILDFLY_STARTUP_TIMEOUT'"/>,g' $WILDFLY_DIR/standalone/configuration/standalone.xml
+        # Enable access from any server
+        sed -i -e 's,<inet-address value="${jboss.bind.address.management:127.0.0.1}"/>,<any-address/>,g' $WILDFLY_DIR/standalone/configuration/standalone.xml
+        sed -i -e 's,<inet-address value="${jboss.bind.address:127.0.0.1}"/>,<any-address/>,g' $WILDFLY_DIR/standalone/configuration/standalone.xml
+
+        # The below line is added to avoid warning when starting WildFly with jdk 8 SE, as the JVM memory parameter changed
+        sed -i -e 's,MaxPermSize,MaxMetaspaceSize,g' $WILDFLY_DIR/bin/standalone.conf
+
+        echo "Configuring Firewalld for WildFly ports"
+        firewall-cmd --permanent --add-port=8080/tcp
+        firewall-cmd --permanent --add-port=8443/tcp
+        firewall-cmd --permanent --add-port=9990/tcp
+        firewall-cmd --permanent --add-port=9993/tcp
+        firewall-cmd --reload
+
+        echo "Backup management user"
+        cp $WILDFLY_DIR/standalone/configuration/mgmt-users.properties $WILDFLY_DIR/standalone/configuration/mgmt-users-org.properties
+        cp $WILDFLY_DIR/standalone/configuration/application-users.properties $WILDFLY_DIR/standalone/configuration/application-users-org.properties
+        cp $WILDFLY_DIR/domain/configuration/mgmt-users.properties $WILDFLY_DIR/domain/configuration/mgmt-users-org.properties
+        cp $WILDFLY_DIR/domain/configuration/application-users.properties $WILDFLY_DIR/domain/configuration/application-users-org.properties
+        chown -R $WILDFLY_USER:$WILDFLY_USER $WILDFLY_DIR/standalone/configuration/mgmt-users-org.properties
+        chown -R $WILDFLY_USER:$WILDFLY_USER $WILDFLY_DIR/standalone/configuration/application-users-org.properties
+        chown -R $WILDFLY_USER:$WILDFLY_USER $WILDFLY_DIR/domain/configuration/mgmt-users-org.properties
+        chown -R $WILDFLY_USER:$WILDFLY_USER $WILDFLY_DIR/domain/configuration/application-users-org.properties
+
+        echo "Starting Wildfly"
+        service $WILDFLY_SERVICE start
+        chkconfig --add wildfly
+        chkconfig --level 2345 wildfly on
+
+        echo "Done."
+        ~~~
+
+2.  Make the script executable:
+
+        sudo chmod +x /opt/wildfly-install.sh
+
+3.  Run the script:
+
+        cd /opt
+        sudo ./wildfly-install.sh
+ 
+4.  You will need to add to add management user (Web and/or CLI) to be able to access the management console, this can be simply done by running the command simply run the shell script in `/opt/wildfly/bin/add-user.sh`:
+
+    1.  Select option a for Management user.
```

Comment:
I assume your comment on the PR thread was a reply to this in-line comment. I'll quote it here for consistency:

```
@alexfornuto 
The steps are commands to be executed not terminal output.
```

If you're going to keep this section, please use some additional formatting (punctuation or bold for the questions versus answers) to better differentiate what's being asked by the script and what you're suggesting as an answer.